### DISCUSSION TITLE
Help with understanding Kimai Functions available

### DISCUSSION THREAD

**Discussion by greavette** [kimai/kimai#3958] [Questions & Answers]
Hello,

I'm interested in finding a self-hosted time-tracking software for our hourly employees to replace our punch card clock.  Our needs are very simple.  I'd like to make it as easy for our employees to use as possible.  I'm looking to have a kiosk tablet set up on a table where the employees can log in with a simple 4-6 digit code (no email and password as that takes too long to type in on a tablet).  Once the employee logs into the app, they simply press a Check In Button on the tablet and then log out.  At the end of their workday, they log in again and Check Out.  Each pay period our bookkeeper would export from the time tracking app each employee's time and import this into our Quickbooks Online account.  

The other function I'm looking to implement is to have our office manager log in to the time-tracking app to record an employee's absence.  So if an employee calls in sick our office manager can record this absence.  The lab manager would want to be able to create reports on employees and their absences.  

As I said my requirements are fairly simple.  Can anyone suggest of Kimai be set up in the way that I need to use it?  I'm not seeing a mobile app or page I can use to mimic the employee logging into a minimal page to simply check in and check out.  

Thank you in advance for any advice you can provide me.

**Comment by kevinpapst** [comment]
Absence & Sickness can be tracked through dedicated projects and then use the project-detail report for detailed analysis.

For the login you can use the Kiosk plugin. It has a mode to use own codes:
https://www.kimai.org/store/kiosk-barcode-bundle.html

So on your tablet the user only has a text input field and enters his code.
Upon submit the user is either logged-in (if not yet tracking) or logged-out (with statistics about the current week).

It can be tested in the Plugin Demo (linked on the shop page above).
